utils/utils.jsx

```diff
@@ -404,8 +404,13 @@ export function isGIFImage(extin) {
     return extin.toLowerCase() === Constants.IMAGE_TYPE_GIF;
 }
 
-export function getFileType(extin) {
-    var ext = extin.toLowerCase();
+const removeQuerystringOrHash = (url) => {
```

Comment:
we don't actually currently support hashes in image links, but I thought it would be best to handle them here anyway